### PR TITLE
[IMP] calendar: improve crowded meeting popover design

### PR DESCRIPTION
**Pull request by leonardopavanrocha**
When meetings are created for many people and with a lot of information, the event pop-up window becomes difficult to read. This PR aims to improve the popover layout for crowded meetings.

task-2685440

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/calendar/static/src/scss/calendar.scss [hunk 1]
```diff
@@ -74,6 +74,27 @@
     padding-top: 0px !important;
 }
 
+.o_cw_popover {
```
Comment (mcm-odoo) [inline addons/calendar/static/src/scss/calendar.scss:77]: These new rules affected the popover of the year view

File: addons/web/static/src/legacy/scss/web_calendar.scss [hunk 2]
```diff
@@ -796,14 +796,6 @@ $o-cw-filter-avatar-size: 20px;
...
         overflow: auto;
         padding-bottom: 1px; // prevents the scrollbar to show when not needed
-
-        &::-webkit-scrollbar {
-            background: gray('200');
-            width: 6px;
-        }
-        &::-webkit-scrollbar-thumb {
-            background: gray('500');
-        }
```
Comment (mcm-odoo) [inline addons/web/static/src/legacy/scss/web_calendar.scss:806]: Why do you move this into calendar?
other apps could benefit from this

**Review by mcm-odoo — COMMENTED** [review]
Hello @leonardopavanrocha and @arj-odoo 
I left two comments.
I think we can have only one change in the framework (for the icon alignment)
Also, I see that there are no tests, I would suggest to write some but the calendar app is your scope.

**Comment by leonardopavanrocha** [comment]
Hello @mcm-odoo, thanks for the review! I updated the code and wrote the tests you required. Could you please check it?

**Review by mcm-odoo — APPROVED** [review]

**Comment by brboi** [comment]
inactive PR, feel free to rebase and reopen

**Pull request closed without merging**